package ch.uzh.ifi.seal.soprafs18.entity;

import ch.uzh.ifi.seal.soprafs18.repository.PlayerRepository;
import org.junit.Test;

import static org.junit.Assert.*;

public class PlayerTest {

	private PlayerRepository playerRepo;

	Player p1 = new Player();
	Player p2 = new Player();
	Player p3 = new Player();
	Player p4 = new Player();
	Player p5 = new Player();
	Player p6 = new Player();

	@Test
	public void getId() throws Exception {
	}

	@Test
	public void getToken() throws Exception {
	}

	@Test
	public void getName() throws Exception {
	}

	@Test
	public void getColor() throws Exception {
	}

	@Test
	public void getDeck() throws Exception {
	}

	@Test
	public void getHand() throws Exception {
	}

	@Test
	public void getMoveCounter() throws Exception {
	}

	@Test
	public void getPlayerLeft() throws Exception {
	}

	@Test
	public void getIsInGoal() throws Exception {
	}

	@Test
	public void getDiscardPile() throws Exception {
	}

	@Test
	public void getPlayedList() throws Exception {
	}

	@Test
	public void setName() throws Exception {
	}

	@Test
	public void playCard() throws Exception {
	}

	@Test
	public void move() throws Exception {
	}

	@Test
	public void buyCard() throws Exception {
	}

	@Test
	public void sellCard() throws Exception {
	}

	@Test
	public void drawCard() throws Exception {
	}

	@Test
	public void addCardToDeck() throws Exception {
	}

	@Test
	public void addCardToDiscardPile() throws Exception {
	}

	@Test
	public void addCardToPlayingList() throws Exception {
	}

	@Test
	public void removeCardFromHand() throws Exception {
	}

	@Test
	public void removeCardFromDiscardPile() throws Exception {
	}

	@Test
	public void removeCardFromDeck() throws Exception {
	}

	@Test
	public void resetPlayedList() throws Exception {
	}

	@Test
	public void resetDiscardPile() throws Exception {
	}

	@Test
	public void setMoveCounter() throws Exception {
	}

	@Test
	public void setCoins() throws Exception {
	}

	@Test
	public void setPlayerLeft() throws Exception {
	}

	@Test
	public void setIsInGoal() throws Exception {
	}

}